src/views/Index.js

```diff
@@ -15,322 +15,182 @@
 * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
 
 */
-import React, { useState } from "react";
+import React, { useEffect, useState, useMemo } from "react";
 // node.js library that concatenates classes (strings)
 import classnames from "classnames";
 // javascipt plugin for creating charts
 import Chart from "chart.js";
 // react plugin used to create charts
 import { Line, Bar } from "react-chartjs-2";
 // reactstrap components
-import {
-  Button,
-  Card,
-  CardHeader,
-  CardBody,
-  NavItem,
-  NavLink,
-  Nav,
-  Progress,
-  Table,
-  Container,
-  Row,
-  Col,
-} from "reactstrap";
 
-// core components
-import {
-  chartOptions,
-  parseOptions,
-  chartExample1,
-  chartExample2,
-} from "variables/charts.js";
+import { Button, Card, CardHeader, CardBody, NavItem, NavLink, Nav, Progress, Table, Container, Row, Col, Input, CustomInput, Modal, ModalHeader, ModalBody, ModalFooter, Form, FormGroup, Label, FormText } from "reactstrap";
 
 import Header from "components/Headers/Header.js";
+import TableContainer from './TableContainer.js'
 
 const Index = (props) => {
-  const [activeNav, setActiveNav] = useState(1);
-  const [chartExample1Data, setChartExample1Data] = useState("data1");
+  // to store the fetched data
+  const [fetchedData, setFetchedData] = useState([])
+  const [isModal, setIsModal] = useState(false)
 
-  if (window.Chart) {
-    parseOptions(Chart, chartOptions());
+  useEffect(() => {
+    // to set the fetched data
+    fetch("https://mockend.com/h4rSHp/fake-api/posts")
+      .then(response => response.json())
+      .then(data => {
+        setFetchedData(data)
+      })
+      .catch(error => console.log(error))
+  }, [])
+
+  const toggle = () => setIsModal(!isModal);
+
+  const renderRowSubComponent = (fetchedData, cells) => {
+    let index = parseInt(cells[0]['row']['id'])
+    let headingstyle = { fontSize: '20px' }
+    return (
+      <Card style={{ margin: "0 auto", color: 'white', background: 'linear-gradient(87deg, #11cdef, #1171ef)' }}>
```

Comment:
Maybe improve the background color. White would look descent enough. And try and match the font styles.